otx/algorithms/classification/configs/efficientnet_b0_cls_incr/compression_config.json

```diff
@@ -3,7 +3,7 @@
     "find_unused_parameters": true,
     "nncf_config": {
       "compression": [],
-      "log_dir": "."
+      "log_dir": "/tmp"
```

Comment:
Are there any other issues with modifying this to `tmp`?

Reply:
For my best knowledge, no. Anomaly templates already have `log_dir` as `/tmp`.